.github/workflows/gradle_publish.yml

```diff
@@ -0,0 +1,104 @@
+# This workflow creates a signed Android Bundle which can be uploaded to Play Store Console
```

Comment:
I noticed that the publish workflow on Bitrise was not that complicated after all, so I moved that to Github Actions, too, as there are now Actions for the steps we used so far.

I only changed that the `versionName` is now also set automatically by the CI, so we don't have to do this manually via a commit.